bundles/org.eclipse.mita.program/src/org/eclipse/mita/program/inferrer/StaticValueInferrer.xtend

```diff
@@ -79,6 +138,10 @@ class StaticValueInferrer {
 	}
 	
 	static dispatch def Object infer(ElementReferenceExpression expression, (EObject) => void inferenceBlockerAcceptor) {
+		val ref = expression.reference;
+		if(ref !== null) {
+			return infer(ref, expression, inferenceBlockerAcceptor);
+		}
 		return expression.reference?.infer(inferenceBlockerAcceptor);
```

Comment:
ref is not null here, so this is a bit shorter and valid now:
`return infer(ref, inferenceBlockerAcceptor)`

Reply:
No, `ref` is `null` here, so returning `null` is the way to go actually